test/uk/gov/hmrc/cdsreimbursementclaim/connectors/FileUploadConnectorSpec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.connectors
+
+import org.scalamock.matchers.ArgCapture.CaptureOne
+import play.api.test.Helpers.{await, _}
+import uk.gov.hmrc.cdsreimbursementclaim.controllers.BaseSpec
+import uk.gov.hmrc.cdsreimbursementclaim.models.Error
+import uk.gov.hmrc.http.logging.Authorization
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class FileUploadConnectorSpec extends BaseSpec with HttpSupport {
+
+  val connector                  = new DefaultFileUploadConnector(mockHttp, appConfig)
+  val backEndUrl                 = "http://localhost:7502/filetransfer/init/1.0.0"
```

Comment:
incorrect URL

Reply:
The pdf has this as the enpoint. Richard will get back to me about it